_includes/projectlist.html

```diff
@@ -2,8 +2,8 @@
   <h3>Projects currently in this phase:</h2>
   <ul>
   {% for project in site.data.projects %}
-    {% if project[1].stage == include.stage %}
-    <li><a href="{{site.baseurl}}/project/{{project[0]}}">{{ project[1].project }}</a></li>
+    {% if project[1]['stage'] == include.stage %}
```

Comment:
I believe you can still use `project[1].stage` here and use `project[1].full_name` below, if you want. Thoughts?